source/python/gui/source/gui.py

```diff
@@ -82,36 +79,47 @@ def build_line_graph(data, KernelName, numPoints):
     layout2 = html.Div(
         id="graph_select",
         className="graph_select",
-        children=[
-            html.H4("Call Stack Sample Histogram", style={"color": "white"}),
-        ],
+        children=[html.H4("Call Stack Sample Histogram", style={"color": "white"})],
     )
 
     return layout1, layout2
 
 
-def update_line_graph(sortFilter, func_list, exp_list, data, numPoints, samples):
+def update_line_graph(
+    sort_filter, func_list, exp_list, data, num_points, samples, workloads
+):
     # df = px.data.gapminder() # replace with your own data source
-    if "Alphabetical" in sortFilter:
+    if "Alphabetical" in sort_filter:
```

Comment:
Aren't these if/elif? If you cannot sort by both Alphabetical and Impact, etc. using if every time instead of if/elif adds unnecessary branching checks